core/test/CMakeLists.txt

```diff
@@ -83,39 +83,59 @@ if(ENABLE_MPI)
         target_compile_definitions(testXiosCalendar_MPI1 PRIVATE USE_XIOS)
         target_include_directories(
             testXiosCalendar_MPI1
-            PRIVATE "${MODEL_INCLUDE_DIR}" "${XIOS_INCLUDE_LIST}" "${ModulesRoot}/StructureModule"
+            PRIVATE
+                ${PHYSICS_INCLUDE_DIRS}
```

Comment:
I had to add these includes to each of the XIOS tests to avoid linking errors when including `Model.hpp`.